supporter-product-data/src/test/scala/com/gu/services/DynamoDBServiceSpec.scala

```diff
@@ -0,0 +1,24 @@
+package com.gu.services
+
+import com.gu.model.Stage.DEV
+import com.gu.model.dynamo.SupporterRatePlanItem
+import com.gu.test.tags.annotations.IntegrationTest
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.time.LocalDate
+
+@IntegrationTest
+class DynamoDBServiceSpec  extends AsyncFlatSpec with Matchers {
+
+  "DynamoDBService" should "stream be able to insert an item" in {
+    val service = DynamoDBService(DEV)
+    val item = SupporterRatePlanItem(
+      "999999",
+      None,
+      "1", "1", "Digital Subscription", LocalDate.parse("2021-01-01")
+    )
+    service.writeItem(item).map(_ => succeed)
```

Comment:
this will pass as long as an exception wasn't thrown. worth to check that, if not rename to 
`"DynamoDBService" should "not throw an exception on insert"`

Reply:
Now checks response was a success